src/axom/inlet/Field.cpp

```diff
@@ -43,7 +21,10 @@ std::shared_ptr<Field> Field::required(bool isRequired)
   {
     std::string msg = fmt::format("Inlet Field has already defined required value: {0}",
                                   m_sidreGroup->getName());
-    issueWarning(msg, m_sidreRootGroup);
+    SLIC_WARNING(msg);
+    if (!m_sidreRootGroup->hasView("warningFlag")) {
```

Comment:
Thanks@aesha for moving this back so we still get the file/line number functionality.  Can you move these three lines into an inline private function so its more of a middle ground between what is now here and what you had before?  Just the setting the warningFlag logic.

Reply:
Maybe this first function could be a new internal inlet helper function file. inlet_utils.hpp?  There was a couple other things that could go into a file like that but it escapes me atm.